build.sbt

```diff
@@ -59,7 +59,7 @@ val catsVersion = "2.9.0"
 val jawnVersion = "1.4.0"
 val shapelessVersion = "2.3.10"
 val refinedVersion = "0.9.29"
-val refinedNativeVersion = "0.10.1"
+val refinedNativeVersion = "0.10.3"
```

Comment:
FYI this change is not binary-breaking :)

Reply:
the description is confusing then 😆 
> from 0.9.29 to 0.10.3